devel/simple-websocket-server/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup cmake     1.1
 
 name                simple-websocket-server
-version             2.0.0-rc4
+version             2.0.0
```

Comment:
MacPorts does not know what "2.0.0" should be considered to be newer than "2.0.0-rc4" so the `epoch` must be increased. Since this port does not yet list an epoch, you would add the line `epoch 1` above the `version` line.

Reply:
did that. Do I need to squash the commits? Also, build fails, but it seems to fail randomly?

Reply:
I can squash the commits. I'll fix the whitespace issues too.

The build failures can be ignored; they're caused by https://trac.macports.org/ticket/60590.